core/controllers/suggestion.py

```diff
@@ -342,13 +350,49 @@ def get(self, target_type, suggestion_type):
             target_type: str. The type of the suggestion target.
             suggestion_type: str. The type of the suggestion.
         """
+        topic_name = self.request.get('topic_name', None)
+
         self._require_valid_suggestion_and_target_types(
             target_type, suggestion_type)
+
+        opportunity_summary_exp_ids = []
```

Comment:
Maybe indicate in the variable name that these are the opportunity summary exp IDs that correspond to the topic.

Reply:
Done